api/v1/storagecluster_types.go

```diff
@@ -133,6 +136,14 @@ type ManageCephBlockPools struct {
 	DisableSnapshotClass bool   `json:"disableSnapshotClass,omitempty"`
 }
 
+// ManageReplica1CephBlockPools defines how to reconcilea CephBlockPools
+type Replica1CephBlockPoolsSpec struct {
```

Comment:
How about just name it for replica 1 so other settings besides pools can be added in the future
```suggestion
type Replica1Spec struct {
```